### PR TITLE
Adding command for loading completion extension

### PR DESCRIPTION
**Pull request by shengyfu**

### CODE REVIEW

File: package.nls.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
 "extension.showGettingStarted":"Getting Started Guide",
 "extension.newQuery":"New Query",
 "extension.rebuildIntelliSenseCache":"Refresh IntelliSense Cache",
+"mssql.loadCompletionExtension":"Load completion extension into sql language service",
```
Comment (kburtram) [inline package.nls.json]: if this text will be show to user we should capitalize `SQL`.

File: package.json [hunk 2]
```diff
@@ -211,6 +212,11 @@
         "command": "extension.rebuildIntelliSenseCache",
         "title": "%extension.rebuildIntelliSenseCache%",
         "category": "MS SQL"
+      },
+      {
+        "command": "mssql.loadCompletionExtension",
+        "title": "%mssql.loadCompletionExtension%",
+        "category": "MS SQL"
```
Comment (kburtram) [inline package.json]: does this show up in the Command Palette?

File: src/models/contracts/languageService.ts [hunk 3]
```diff
@@ -111,3 +111,27 @@ export class DidChangeLanguageFlavorParams {
...
+    /// </summary>
+    public assemblyPath: string;
+    /// <summary>
+    /// The type name for the completion extension
+    /// </summary>
+    public typeName: string;
+    /// <summary>
+    /// Property bag for initializing the completion extension
+    /// </summary>
+    public properties: {};
```
Comment (kburtram) [inline src/models/contracts/languageService.ts]: we don't have any better type here?

File: package.json (hunk 2 above)
Comment (shengyfu) [reply package.json]: This is just for testing, it's removed now.

File: package.nls.json (hunk 1 above)
Comment (shengyfu) [reply package.nls.json]: No need to expose the command in Command Palette, removed now.

File: src/models/contracts/languageService.ts (hunk 3 above)
Comment (shengyfu) [reply src/models/contracts/languageService.ts]: This is to match the properties dictionary needed for initializing a CompletionExtension.

**Pull request merged**